src/game/Background.ts

```diff
@@ -0,0 +1,16 @@
+export class Background {
+  height: number;
+  widht: number;
```

Comment:
❗
И везде, где такая же опечатка
```suggestion
  width: number;
```